src/components/comment-form/comment-form.tsx

```diff
@@ -0,0 +1,138 @@
+import { useState, ChangeEvent } from 'react';
+interface FormType {
+  rating: string | null;
+  review: string;
+}
+function CommentForm() {
+  const [form, setForm] = useState<FormType>({
+    rating: null,
+    review: '',
+  });
+  const ratingChangeHandler = (e: ChangeEvent<HTMLInputElement>) => {
+    setForm({
+      ...form,
+      rating: e.target.defaultValue
```

Comment:
вкусовщина, но мне кажется не очень хорошо смешивать реактовские абстракции с браузерными событиями
```suggestion
      rating: e.target.value
```